lib/apartment/railtie.rb

```diff
@@ -27,7 +27,7 @@ class Railtie < Rails::Railtie
     #   See the middleware/console declarations below to help with this. Hope to fix that soon.
     #
     config.to_prepare do
-      unless ARGV.any? { |arg| arg =~ /\Aassets:(?:precompile|clean)\z/ }
+      unless ARGV.any? { |arg| arg =~ /\Aassets:(?:precompile|clean)\z|\Adb:create\z/ }
```

Comment:
Also need to handle `db:setup`

Reply:
Oops, yes. We need it too. But there is no guarantee that it is sufficient... 🤔